src/Polly.Pcl/project.json

```diff
@@ -1,12 +1,7 @@
 ﻿{
-    "dependencies": {
-        "NuSpec.ReferenceGenerator": {
-            "version": "1.4.0",
-            "type": "build"
-        }
-    },
-    "frameworks": {
-        ".NETPortable,Version=v4.5,Profile=Profile259": { }
-    },
-    "supports": {}
-}
+  "dependencies": {},
+  "frameworks": {
+    ".NETPortable,Version=v4.5,Profile=Profile259": {}
```

Comment:
Thanks @twsouthwick .  Yes, we're removing the PCL259 target in favour of the .NET Standard 1.0 target, as part of shortly-forthcoming Polly v5.0 RTM.  We figured last summer this was the way to go, but were deep in the many new features for Polly v5.0, and unsure at that time which would be the lowest .NET Standard we'd be able to target.  

(Looking forward to tidying up the solution further when Visual Studio "15" / 2017 is out at RTM / with stable project migration. I see from the [Visual 2017 RC release notes](https://www.visualstudio.com/en-us/news/releasenotes/vs2017-relnotes) couple days ago that the migration tooling is still at preview.)

Thanks @twsouthwick for all your input and support with this.